content/product-engineering/engineering/cloud/devops/continuous_delivery.md

```diff
@@ -0,0 +1,13 @@
+# Sourcegraph Cloud Continuous Delivery
+
+This is a brief overview of the CD process for Sourcegraph Cloud (sourcegraph.com).
+
+1. New commits pushed to the [mono repo](https://github.com/sourcegraph/sourcegraph) trigger builds by `BuildKite`. The pipelines are all generated [from code](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@main/-/tree/enterprise/dev/ci). To learn more about the build pipelines and the methodology you can read the documentation on [Continuous Integration](https://docs.sourcegraph.com/dev/background-information/continuous_integration).
+   - Docker images are built by [BuildKite](https://buildkite.com/sourcegraph/sourcegraph) on commit to `main`. These images are pushed to [`GCR`](https://console.cloud.google.com/gcr/images/sourcegraph-dev?project=sourcegraph-dev) with the tag format `<commit-hash>_<build-number>_candidate`.
+   - Buildkite executes the [`sourcegraph-upgrade`](https://github.com/sourcegraph/sourcegraph/blob/main/.buildkite/vagrant-run.sh) build step in which it spins up a VM to test the build starts correctly.
+   - Buildkite executes [puppeteer tests](https://github.com/sourcegraph/sourcegraph/blob/main/enterprise/dev/ci/internal/ci/operations.go#L178) to test front-end behavior.
+   - If both test suites pass the images are "promoted" and pushed to DockerHub with the tag format `<build-number>_<date>_<commit-hash>`.
```

Comment:
@sourcegraph/dev-experience 
Could you all confirm the above? Or is there a place we should link to? 
@danieldides might be worth adding a line here to say that this is an area of dev-experience "guardianship" 
We are primarily focused on the steps after this. 
Where CI meets CD.

Reply:
Good call. I just made a comment and linked (I hope) to the dev-ex team on.

Reply:
```suggestion
   - If all pipeline steps pass the images are "promoted" and pushed to Docker Hub with the tag format `<build-number>_<date>_<commit-hash>`.
```